### ISSUE TITLE
Where is DeviceInfo.h?

### ISSUE THREAD

**Issue by GlennWood** [GPUOpen-Tools/GPA#17]
... and other source files for ../../Src/DeviceInfo/?

    root@rig-AMD:/opt/GPA/Build/Linux# bash ./build.sh 
    Build GPUPerfAPI, 64-bit...
    Failed to build GPUPerfAPI, 64 bit
    
    root@rig-AMD:/opt/GPA/Build/Linux# tail GPUPerfAPI_Build.log 
    In file included from ../GPUPerfAPI-Common/IGPAContext.h:12:0,
                 from ../GPUPerfAPI-Common/GPAContext.h:17,
                 from ../GPUPerfAPI-Common/GPAPass.h:18,
                 from ../GPUPerfAPI-Common/IGPACommandList.h:13,
                 from ../GPUPerfAPI-Common/GPUPerfAPI.cpp:25:
    ../GPUPerfAPI-Common/GPAHWInfo.h:12:24: fatal error: DeviceInfo.h: No such file or directory
    compilation terminated.
    ../../Build/Linux/CommonTargets.mk:56: recipe for target '../../Output/obj/release/x64/GPUPerfAPI      /GPUPerfAPI.o' failed
    make: *** [../../Output/obj/release/x64/GPUPerfAPI/GPUPerfAPI.o] Error 1
make: Leaving directory '/opt/GPA/Src/GPUPerfAPI'

**Comment by chesik-amd** [comment]
It sounds like you forgot to run the UpdateCommon.py script. This script will clone/pull all the dependent repos required by GPA.

This is described here:  https://github.com/GPUOpen-Tools/GPA/blob/master/BUILD.md#cloningupdating-dependent-repositories

**Comment by GlennWood** [comment]
You are right. I had simply skipped down to the "Linux Build Information" section.

Isn't this kind of cloning customarily done with git submodules? That would make for more standard installation instructions; no need for UpdateCommon.py script at all (after you integrate Vulcan installation into build.sh).

I have now completed the build. Thanks!

BTW: BUILD.md reads "cd into the Build\Linux directory", which should be "cd into the Build/Linux directory".
